examples/js/loaders/SVGLoader.js

```diff
@@ -1160,7 +1263,7 @@ THREE.SVGLoader.pointsToStrokeWithBuffers = function () {
 		// if 'vertices' parameter is undefined no triangles will be generated, but the returned vertices count will still be valid (useful to preallocate the buffers)
 		// 'normals' and 'uvs' buffers are optional
 
-		arcLengthDivisions = arcDivisions !== undefined ? arcDivisions : 12;
+		arcDivisions = arcDivisions !== undefined ? arcDivisions : 12;
```

Comment:
@yomboprime FYI: This line cause a runtime error with the module version since `arcLengthDivisions` is not defined.

Reply:
Would `var arcLengthDivisions = arcDivisions !== undefined ? arcDivisions : 12;` solve it?

Reply:
Yes, that would have also worked. I have now just used `arcDivisions` similar to the other parameters.

Reply:
> Yes, that would have also worked. I have now just used `arcDivisions` similar to the other parameters.

Yes, sorry, that was a var renaming typo.

Reply:
Sounds good!